src/main/webapp/index.jsp

```diff
@@ -32,7 +32,9 @@
       <a href="/register">Register</a>
     <% } %>
     <a href="/about.jsp">About</a>
-    <a href="/testdata">Load Test Data</a>
+    <% if(request.getSession().getAttribute("admin") != null){ %>
+      <a href="/admin">Admin</a>
+    <% } %>
```

Comment:
This will not show up on all of our pages. Something we should address when we make a common navigation bar/ UI update.